geth/peers/signal.go

```diff
@@ -0,0 +1,48 @@
+package peers
+
+import (
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/status-im/status-go/geth/signal"
+)
+
+const (
+	// DiscoveryStarted is sent when discv5 server was started.
+	DiscoveryStarted = "discovery.started"
+	// DiscoveryStopped is sent when discv5 server was stopped.
+	DiscoveryStopped = "discovery.stopped"
+
+	// DiscoverySummary is sent when peer is added or removed.
+	// it will be a map with capability=peer count k/v's.
+	DiscoverySummary = "discovery.summary"
+)
+
+// DiscoverySignals defines handlers for sending discovery related signals.
+type DiscoverySignals struct{}
+
+// DiscoveryStarted sends discovery.started signal.
+func (d DiscoverySignals) DiscoveryStarted() {
```

Comment:
It's fine but these could be regular functions.

Reply:
you are right, i wanted at first to make sending signal optional. will simplify